sh/update.sh

```diff
@@ -11,4 +11,5 @@ aws_secret_access_key=${AWS_SECRET_ACCESS_KEY} \
 cluster_name=${CLUSTER_NAME} \
 platform=${PLATFORM} \
 cluster_environment=${CLUSTER_ENVIRONMENT} \
+oidc_issuerurl=${OIDC_ISSUER_URL} \
```

Comment:
Please do not merge this until we also update the Jenkins job for updating a cluster

Reply:
I managed to update the pipeline library of the job and it is using ATM the version that is able to handle the new parameter.